airflow-core/src/airflow/cli/cli_config.py

```diff
@@ -599,6 +599,13 @@ def string_lower_type(val):
     default=False,
 )
 
+ARG_DB_MANAGER_NAME = Arg(
+    ("--name",),
```

Comment:
```suggestion
    ("name",),
```

Having it be positional might be a bit nicer?